mock/api.js

```diff
@@ -13,34 +13,19 @@ export const mockControllers = new Array(DEFAULT_SIZE).fill().map((value, index)
   results: casual.integer(1, DEFAULT_SIZE),
 }));
 
-export const mockResults = {
-  took: DEFAULT_SIZE,
-  hits: {
-    total: {
-      value: DEFAULT_SIZE,
-    },
-    hits: new Array(DEFAULT_SIZE).fill().map((value, index) => ({
-      _index: moment().format('YYYY-MM'),
-      _type: casual.word,
-      _id: casual.uuid,
-      _score: null,
-      _source: {
-        '@metadata': {
-          controller_dir: casual.word,
-        },
-        run: {
-          controller: casual.word + index,
-          name: casual.word,
-          start: moment.utc() + index,
-          end: moment.utc() + index,
-          id: casual.uuid,
-          config: casual.word,
-        },
-      },
-      sort: [casual.unix_time],
-    })),
-  },
-};
+export const mockResults = new Array(DEFAULT_SIZE).fill().map((value, index) => ({
+  '@metadata.controller_dir': casual.word,
+  key: casual.uuid,
+  id: casual.uuid,
+  config: casual.word,
+  controller: casual.word + index,
+  deletion: moment.utc().add('days', 30),
```

Comment:
Several of your fields are actually requests for changes. Neither `deletion`, `seen`, `saved`, nor `status` has any precedent. This is fine, but best to defer these for a later PR. We need to add them on the server side, and the mocks should be mocking what the server returns!

Reply:
Removed these. Will add these filed in overview page PR #105 
(these fields will be needed there for data rendering)